src/app/common/app.helpers.ts

```diff
@@ -56,3 +60,8 @@ export function momentConvertDatetoLocalWithFormat(dateParams: {
 export function parseCurrentSchema(data: MaybeNullOrUndefined<DataSchema>): MaybeNull<DatasetSchema> {
     return data ? (JSON.parse(data.content) as DatasetSchema) : null;
 }
+
+export function nextTime(value: string): string {
```

Comment:
Looking at function name and argument name I would have never guessed this is about CRON expressions.

Reply:
Renamed to `cronExpressionNextTime`